.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@ repos:
         # supported by your project here, or alternatively use
         # pre-commit's default_language_version, see
         # https://pre-commit.com/#top_level-default_language_version
-        language_version: python3.9
+        language_version: python3.11
```

Comment:
can you revert this to 3.9 so that the precommit passes

Reply:
Yes, I just had to do this locally to somehow get through my pre-commits in order to push this...